tensorflow/lite/experimental/acceleration/configuration/configuration.proto

```diff
@@ -570,6 +570,11 @@ message CPUSettings {
   optional int32 num_threads = 1 [default = -1];
 }
 
+// Arm NN delegate option
+message ArmNNSettings {
+  optional string delegate_options = 1;
```

Comment:
Could you please document this field, or include a pointer to where developers can find documentation for it?

Reply:
Hi @fergushenderson Could you please review it again ? The comment has been added. Sorry for the delay.